samples/client/src/main/java/global/covesa/sdk/client/ui/MainUi.kt

```diff
@@ -73,10 +81,51 @@ fun MainUi(viewModel: MainViewModel) {
         ) {
             Text(text = "Set internal lights")
         }
-
         if (lightsUiState.installedServiceVersion is LightServiceVersion.Installed) {
             LightStates(lightsUiState.lights)
         }
+
+        HorizontalDivider(
```

Comment:
On a smaller screen, this content does not fit, maybe make the whole screen scrollable with `Scaffold`?